src/theme/FileUploader.ts

```diff
@@ -0,0 +1,24 @@
+import { mode } from "@chakra-ui/theme-tools"
+
+export const FileUploader = {
```

Comment:
hmm my issue here is that I don't have an anatomy object to use because chakra doesn't have a file upload component.

I wonder if I could create my own using something like:
```
import { Anatomy } from "@chakra-ui/theme-tools"

export declare const fileUploadAnatomy: Omit<
  Anatomy<"container" | "uploadArea">,
  "parts"
>
```

but I'm not 100% sure how the new type declaration & import will work with storybook. Any ideas?

Reply:
I see but can't we create as in other places? eg:
```
const parts = anatomy("SecondaryAnnouncementBanner").parts(
  "container",
  "closeButton",
  "image",
  "subContainer",
  "title",
  "subTitle",
  "ctaButton"
)
```

Reply:
Fixed